specification/appconfiguration/resource-manager/readme.md

```diff
@@ -26,7 +26,16 @@ These are the global settings for the AppConfiguration API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-11-01-preview
+tag: package-2020-06-01
+```
+
+### Tag: package-2020-06-01
```

Comment:
Please add this tag to the `readme.*.md` for which language you want to generate sdk. Make sure sdk generation could all pass.

Reply:
@ChenTanyi 
I added it into the GO readme. Looks like it passed. Please merge, thanks.